eng/testing/tests.ioslike.targets

```diff
@@ -25,8 +25,7 @@
     <_AfterBuildCommands>
       mv $XHARNESS_OUT/AOTBuild.binlog &quot;$HELIX_WORKITEM_UPLOAD_ROOT&quot;
       sign &quot;$app&quot;
-      xharness apple test --app &quot;$app&quot; --output-directory &quot;$output_directory&quot; --target &quot;$target&quot; --timeout &quot;$timeout&quot; --xcode &quot;$xcode_path&quot; -v --launch-timeout &quot;$launch_timeout&quot; $(_ResetSimulatorSwitch) $(_SignalAppEndSwitch) -- 
-    </_AfterBuildCommands>
+      xharness apple test --app &quot;$app&quot; --output-directory &quot;$output_directory&quot; --target &quot;$target&quot; --timeout &quot;$timeout&quot; --xcode &quot;$xcode_path&quot; -v --launch-timeout &quot;$launch_timeout&quot; $(_ResetSimulatorSwitch) $(_SignalAppEndSwitch) -- </_AfterBuildCommands>
```

Comment:
this is quite brittle, we should at least add a comment why the xml end tag is on the same line.

I'd much prefer if we refactored the code to put together the RunScriptCommand where we actually know all the arguments, rather than hoping nobody appends a newline between here and somewhere else.

Reply:
thinking about it some more, the `RunScriptCommand` is expected to be single-line. we should keep it that way and instead move the things that need to happen before the `xharness test` into a separate ItemGroup which we prepend similar to PreExecutionTestScript in tests.targets

Reply:
so this would become:
```xml
<ItemGroup>
  <PreRunScriptCommands Include="$(_AOTBuildCommand)" />
  <PreRunScriptCommands Include="mv $XHARNESS_OUT/AOTBuild.binlog &quot;$HELIX_WORKITEM_UPLOAD_ROOT&quot;" />
  <PreRunScriptCommands Include="sign &quot;$app&quot;" />
</ItemGroup>
```